tests/Auth0Test.php

```diff
@@ -0,0 +1,152 @@
+<?php
```

Comment:
See [`Add failing test` commit](https://github.com/auth0/auth0-PHP/pull/318/commits) to see initial tests failing with existing functionality.